tube-player/README.md

```diff
@@ -37,29 +40,52 @@ This workshop has been set up to provide you with optional content to allow you
 
 The workshop utilizes the latest Uno Platform features, including:
 
-- [C# Markup](xref:Reference.Markup.GettingStarted)
-- [Extensions](xref:Overview.Features)
-  - [MVUX](xref:Overview.Mvux.Overview)
-    - [Pagination](xref:Overview.Mvux.Advanced.Pagination)
-    - [FeedView](xref:Overview.Mvux.FeedView)
-      - NoneTemplate
-      - ErrorTemplate
-    - [Refit](xref:Overview.Http)
-    - [Region navigation](xref:Overview.Navigation)
-    - [Serialization](xref:Overview.Serialization)
+### Environment setup & Inner loop
+
+- [Hot reload](xref:Uno.Features.HotReload)
+- [Uno.Check](xref:UnoCheck.UsingUnoCheck)
+
+### Cross-platform development
+
+- [Resizetizer](xref:Uno.Resizetizer.GettingStarted)
```

Comment:
Since "Resizetizer" refers to the .NET MAUI one, I think would be good to add the `Uno.` prefix here

```suggestion
- [Uno.Resizetizer](xref:Uno.Resizetizer.GettingStarted)
```